requirements.txt

```diff
@@ -1,30 +1,50 @@
-#
-# Requirements for wger for production
-#
-
-Django>=1.9,<1.10
-django-recaptcha
-reportlab
-django_mobile
-django-formtools>=1.0,<1.1
-bleach
-python-mimeparse
-pillow
-easy-thumbnails
-django_compressor
-icalendar
-django-bootstrap-breadcrumbs
-sphinx
-django-sortedm2m
-django-bower
-invoke
-requests
-
-# REST API
-djangorestframework>=3.2,<3.3
-django-filter
-django-tastypie
-django-cors-headers
-
-# Python3 compatibility
-six
+alabaster==0.7.10
```

Comment:
I recommend you only add the packages you manually installed, their dependancies will automatically be installed, you do not have to add them here.

Reply:
I will work on that.